packages/devtools_app/lib/src/screens/inspector_v2/inspector_tree_controller.dart

```diff
@@ -990,9 +990,12 @@ class _InspectorTreeState extends State<InspectorTree>
       safeViewportHeight,
     );
 
+    final centerLeftHalf = Offset(
```

Comment:
Is it worth leaving some documentation here explaining why we do this and referencing the original issue (or this PR)? ASCII art could work too ;-)

Reply:
Done!